tools/asciidoctor/postprocessor.rb

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+class Copyright < Asciidoctor::Extensions::Postprocessor
+  def process document, output
+    if document.basebackend? 'html'
+      top = %{<!-- Copyright (C) 2020  Matthew Glazar -->
+<!-- See end of file for extended copyright information. -->
+}
+      bottom = %{
+
+<!--
+quick-lint-js finds bugs in JavaScript programs.
+Copyright (C) 2020  Matthew Glazar
+
+This file is part of quick-lint-js.
+
+quick-lint-js is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+quick-lint-js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with quick-lint-js.  If not, see <https://www.gnu.org/licenses/>.
+-->
+<!--
+Other files may have different copyright and may be licensed under different
+licenses. See license.html for details.
+-->
```

Comment:
Nit: This comment isn't needed, since we're only including stuff under quick-lint-js's license and copyright. (I only include this notice when embedding images from other projects.)

Reply:
This was an error on my part. I was copy and pasting the copyright info from the website's index.html page.